ambry-api/src/main/java/com/github/ambry/quota/QuotaMetrics.java

```diff
@@ -69,4 +76,15 @@ public QuotaMetrics(MetricRegistry metricRegistry) {
     noChargeRate = metricRegistry.meter(MetricRegistry.name(QuotaManager.class, "NoChargeRate"));
     noRecommendationRate = metricRegistry.meter(MetricRegistry.name(QuotaManager.class, "NoRecommendationRate"));
   }
+
```

Comment:
Nit: Add a comment indicating that this routine by itself is not thread-safe, however, call to this routine is synchronized